tests/make/makeStore.t.ts

```diff
@@ -0,0 +1,417 @@
+// Note from Evan: For some reason, my VSCode won't appropriately assess the return type on keys of store objects. For example, when the input store has a key of *count: number*, the returned store's key is typed *count: unknown*.
+// Isn't an issue for the repo my orginazation is using to test this stuff.
+import { describe, expect, it, fn, beforeEach } from "jest"; // Need import, or should I convert this file to vitest ?
```

Comment:
Vitest for consistency of the repo, make it '.test.ts' as well

Reply:
I'm surprised tests passed with the jest import, tbh :sweat_smile:

Reply:
ah -- they didn't run at all because the file doesn't end with `.test.ts`
![image](https://github.com/user-attachments/assets/c8fee80c-9bb1-4150-b0ae-ce145f7832a3)